mteb/tasks/Classification/kor/KlueTC.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class KlueTC(AbsTaskClassification):
```

Comment:
Any reason why this is not a clustering task?

Reply:
Let us keep it in line with the paper then to allow results to be at least semi-comparable